scss/options_trackers.ts

```diff
@@ -50,6 +50,10 @@ export const initializeTrackers = (resetCssFunc: Function) => {
             options.resetTheme();
             resetCssFunc();
         }
+        if (options.wallpaper.pywal && dependencies('wal')) {
```

Comment:
```suggestion
        if (options.wallpaper.pywal.value && dependencies('wal')) {
```
We have to add `.value` here otherwise it will always evaluate to true! Looks good otherwise!

Reply:
Good catch, got that fixed up now.